### ISSUE TITLE
Screen reader unable to read "{" and "}" in v1.86 and greater

### ISSUE THREAD

**Issue by rohanshardha** [microsoft/vscode#206215]
Since updating to v1.86 my screen reader (JAWS) is not reading "{" and "}" and just says "blank" in their places. Did not had this issue with v1.85.2. 

Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.86.2 
- OS Version: Windows_NT x64 10.0.22631 

Steps to Reproduce:

1. 
2.

**Comment by rohanshardha** [comment]
@meganrogge

**Comment by meganrogge** [comment]
/duplicate [#200956](https://github.com/microsoft/vscode/issues/200956)
